### PR TITLE
GH#51081 - Updating boot_device to include aarch64

### PR DESCRIPTION
**Pull request by kelbrown20**
For versions 4.11+
Fixed issue: #51081

Preview: 
[Installation configuration -> Customizing nodes -> About disk encryption](https://51146--docspreview.netlify.app/openshift-enterprise/latest/installing/install_config/installing-customizing.html#installation-special-config-encrypt-disk_installing-customizing)
[Installation configuration -> Customizing nodes -> Configuring disk encryption and mirroring ](https://51146--docspreview.netlify.app/openshift-enterprise/latest/installing/install_config/installing-customizing.html#installation-special-config-storage-procedure_installing-customizing)

### CODE REVIEW

**Comment by kelbrown20** [comment]
@aleskandro Hey Alessandro! Can you PTAL at this PR relating to issue #51081 when you get the chance? Thank you!

File: modules/installation-special-config-storage.adoc [hunk 1]
```diff
@@ -54,21 +54,22 @@ metadata:
...
         thumbprint: VCJsvZFjBSIHSldw78rOrq7h2ZF
-    threshold: 2 <3>
+    threshold: 2 <4>
 openshift:
   fips: true
 ----
-<1> Include this field if you want to use a Trusted Platform Module (TPM) to encrypt the root file system.
-<2> Include this section if you want to use one or more Tang servers.
-<3> Specify the minimum number of TPM v2 and Tang encryption conditions that must be met for decryption to occur.
+<1> On ppc64le nodes, set this field to `ppc64le`. On arm64 based nodes, set this field to `aarch64`. On all other nodes, this field can be omitted.
```
Comment (aleskandro) [inline modules/installation-special-config-storage.adoc:69]: Hi @kelbrown20, I know 'examples' are not well appreciated in the doc, but the following is to try giving you the concept I'd like to expose here:

```
Set the <arch> field to the instruction set architecture of the cluster nodes: e.g., x86_64, aarch64, ppc64le...
```

with `layour: <arch>` at row 57.

File: modules/installation-special-config-storage.adoc [hunk 2]
```diff
@@ -196,7 +197,7 @@ openshift:
 ----
 +
 <1> For control plane configurations, replace `worker` with `master` in both of these locations.
-<2> On ppc64le nodes, set this field to `ppc64le`.  On all other nodes, this field can be omitted.
+<2> On ppc64le nodes, set this field to `ppc64le`. On arm64 based nodes, set this field to `aarch64`. On all other nodes, this field can be omitted.
```
Comment (aleskandro) [inline modules/installation-special-config-storage.adoc:200]: The same comment as above. I suppose this is an old statement when only ppc64le and x86_64 were available.

**Comment by aleskandro** [comment]
cc @pamoedom

File: modules/installation-special-config-storage.adoc (hunk 1 above)
Comment (aleskandro) [reply modules/installation-special-config-storage.adoc:69]: I'm a bit concerned about how to express that we need the GNU way of expressing the architecture (aarch64, and not arm64) here, while most of the other parts of the docs use the Golang/llvm way (arm64, amd64).

File: modules/installation-special-config-storage.adoc [hunk 3]
```diff
@@ -54,21 +54,22 @@ metadata:
...
         thumbprint: VCJsvZFjBSIHSldw78rOrq7h2ZF
-    threshold: 2 <3>
+    threshold: 2 <4>
 openshift:
   fips: true
 ----
-<1> Include this field if you want to use a Trusted Platform Module (TPM) to encrypt the root file system.
-<2> Include this section if you want to use one or more Tang servers.
-<3> Specify the minimum number of TPM v2 and Tang encryption conditions that must be met for decryption to occur.
+<1> Set this field to the instruction set architecture of the cluster nodes. Some examples include, `x86_64`, `aarch64`, `ppc64le` and others.
```
Comment (kelbrown20) [inline modules/installation-special-config-storage.adoc:69]: Thank you for the feedback @aleskandro! I know what you mean with the example yamls, but I think that this new sentence is more general and can still work with the example `x86_64` in the field. I'm leaning towards this update as well bc we usually want the formating of the sample yaml/ fields to be consistent. What do you think?

Comment (kelbrown20) [reply modules/installation-special-config-storage.adoc:69]: Currently re wording that last sentence.

File: modules/installation-special-config-storage.adoc [hunk 4]
```diff
@@ -54,21 +54,22 @@ metadata:
...
         thumbprint: VCJsvZFjBSIHSldw78rOrq7h2ZF
-    threshold: 2 <3>
+    threshold: 2 <4>
 openshift:
   fips: true
 ----
-<1> Include this field if you want to use a Trusted Platform Module (TPM) to encrypt the root file system.
-<2> Include this section if you want to use one or more Tang servers.
-<3> Specify the minimum number of TPM v2 and Tang encryption conditions that must be met for decryption to occur.
+<1> Set this field to the instruction set architecture of the cluster nodes. Some examples include, `x86_64`, `aarch64`, or`ppc64le`.
```
Comment (kelbrown20) [inline modules/installation-special-config-storage.adoc:69]: Thank you for the feedback @aleskandro! I know what you mean with the example yamls, but I think that this new sentence is more general and can still work with the example `x86_64` in the field. I'm leaning towards this update as well bc we usually want the formating of the sample yaml/ fields to be consistent. What do you think?

Comment (aleskandro) [reply modules/installation-special-config-storage.adoc:69]: Works for me :)

Thanks

**Comment by pamoedom** [comment]
/lgtm
thks

**Comment by kelbrown20** [comment]
QE ack in [this comment](https://github.com/openshift/openshift-docs/pull/51146#discussion_r986043783)
/label merge-review-needed

**Comment by mburke5678** [comment]
/enterprise-4.11

**Pull request merged**